src/Shell/Task/ModelTask.php

```diff
@@ -973,6 +980,16 @@ public function getTable($name)
     }
 
     /**
+     * Get the prefix for model if exist.
+     *
+     * @return string
+     */
+    public function getModelPrefix()
```

Comment:
But does it need to be part of the public API here i wonder? It is only used internally here, right?